src/assets/templates/typescript/src/steps/user-field-equals.ts

```diff
@@ -52,6 +61,7 @@ export class UserFieldEqualsStep extends BaseStep implements StepInterface {
     const email: string = stepData.email;
     const field: string = stepData.field;
     const expectedValue: string = stepData.expectedValue;
+    const operator: string = stepData.operator || 'be'; // Default to 'be' or 'equal to'
```

Comment:
- [ ] Similar comment.  Don't need a default; it was just required on existing cog code for BC.
- [ ] You may want to stick the `.toLowerCase()` on the operator value here though.